scripts/script.js

```diff
@@ -0,0 +1,24 @@
+//Change Fun Facts to a different fact with a set interval
+
+const FACT_CHANGE_INTERVAL = 7000;
```

Comment:
Super minor, but let's put this into an iife

https://en.wikipedia.org/wiki/Immediately-invoked_function_expression